rust/ql/test/query-tests/unusedentities/main.rs

```diff
@@ -189,7 +189,7 @@ enum YesOrNo {
     No,
 }
 
-use YesOrNo::{Yes, No}; // allows `Yes`, `No` to be accessed without qualifiers.
+use YesOrNo::{No, Yes}; // allows `Yes`, `No` to be accessed without qualifiers.
```

Comment:
Why this change?

Reply:
Hmm, I think it's the auto-formatter.

Reply:
In general I believe we should not autoformat test code in the target language.  Autoformatting adds a regularity that could (theoretically) mask issues in the extractor.

Reply:
Not that I strongly care about this particular case (reverting not necessary).